shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js

```diff
@@ -1060,59 +1056,3 @@ describe('serveCaseToIrsInteractor', () => {
     });
   });
 });
-
-describe('addDocketEntryForPaymentStatus', () => {
```

Comment:
Already moved to `serveCaseToIrsInteractor.addDocketEntryForPaymentStatus.test.js`